docs/architecture/grpc-for-wcf-developers/create-project.md

```diff
@@ -23,11 +23,11 @@ Select **Next** to continue to the **Configure your new project** dialog box. Na
 
 Select **Next** to continue to the **Create a new gRPC service** dialog box.
 
-![Screenshot of Create a new gRPC service dialog box](media/create-project/create-new-grpc-service.png)
+![Screenshot of Create a new gRPC service dialog box](media/create-project/create-new-grpc-service-v2.png)
```

Comment:
Was *media/create-project/create-new-grpc-service.png* no longer needed, and if not was it deleted?

Reply:
`create-new-grpc-service.png` - is no more needed. So I have deleted it.